phys2bids/interfaces/txt.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+phys2bids interface for txt files.
+"""
+from bioread import read_file
+
+from phys2bids.physio_obj import BlueprintInput
+
+import numpy as np
+# '/home/vicente/phys2bids/phys2bids/tests/data/Test_multiscan100Hz_trig_samefreq_header.txt'
+def populate_phys_input(filename, chtrig):
+    """
+    Populate object phys_input
+    for now we this works with labchart files
+    """
+    header = []
+    channel_list = []
+    with open(filename,'r') as f:
+        header_l = 0
+        for line in f:
+            line=line.rstrip('\n').split('\t')
+            try: 
+                float(line[0])
+            except ValueError:
+                header.append(line)
+                continue 
+            line = [float(i) for i in line]
+            channel_list.append(line)
+    # get frequency 
+    interval = header[0][1].split(" ")
+    if interval[-1] not in ['hr', 'min', 's','ms','µs']:
+        raise AttributeError(f'Interval is not in a valid LabChart time unit, this probably means your file is not in labchart format')
+
+    if interval[-1] != 's':
+        print(f'interval is not in s converting')
```

Comment:
f-string is not necessary here! You can leave it as a normal string.
```suggestion
        print('Interval is not in seconds. Converting its value.')
```